addons/mail/static/src/js/form_controller.js

```diff
@@ -13,6 +13,25 @@ FormController.include({
         new_message: '_onNewMessage',
     }),
 
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async saveRecord() {
+        const changedFields = await this._super(...arguments);
+        const chatter = this.renderer.chatter;
+        if (chatter) {
+            chatter.trigger_up('reset_suggested_partners');
```

Comment:
should call a public method instead, e.g. `resetSuggestedPartners()`.
`Chatter` should internally handle updating suggested recipients in case the composer is open.

To do so, I would suggest having a private attribute `_isComposerOpen` in `Chatter` that is toggled on/off based on whether composer is open (e.g. `_openComposer` should toggle it on).